CONTRIBUTING.md

```diff
@@ -28,6 +28,10 @@ issues first, as it's possible that someone else reported the same issue before
 you. Though it helps save time, don't worry! We won't mind if you accidentally
 post a duplicate issue.
 
+Please note that if an issue can only be reproduced on a nightly compiler it will be closed and marked "Will not fix."
```

Comment:
I'd say it's the responsibility of the person running the nightly compiler to report that, not ours.  They took the jump into nightly, we didn't.